clients/onboarding/src/utils/exchanges/suspenseDedupExchange.ts

```diff
@@ -0,0 +1,41 @@
+import { Exchange, Operation, OperationResult, makeOperation } from "@urql/core";
```

Comment:
As we have to duplicate this for all clients, it might be interesting to put it in `lake/shared-business`. What do you think?

Reply:
Maybe in a future PR, but we will have to put `requestIdExchange` there too.

Reply:
ok this is good for me to do it later